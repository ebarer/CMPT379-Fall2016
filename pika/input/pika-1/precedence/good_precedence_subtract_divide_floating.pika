exec {

  print 6.9 - 8.8 / 7.7, _n_.
  print 5.9 - 1.3 / 2.6, _n_.
  print 6.8 - 1.0 / 8.2, _n_.
  print 3.1 - 4.7 / 2.8, _n_.
  print 8.6 - 1.2 / 3.0, _n_.
  print 7.6 - 1.4 / 5.5, _n_.
  print 7.4 - 3.5 / 6.1, _n_.
  print 3.5 - 2.6 / 4.0, _n_.
  print 2.5 - 4.2 / 3.0, _n_.
  print 7.2 - 2.7 / 2.6, _n_.
  print 5.4 - 4.9 / 8.7, _n_.
  print 1.4 - 3.2 / 5.1, _n_.

  print 6.6 / 5.1 - 7.1, _n_.
  print 7.1 / 8.0 - 5.3, _n_.
  print 9.2 / 3.6 - 2.0, _n_.
  print 6.2 / 4.3 - 6.5, _n_.
  print 4.5 / 1.4 - 8.8, _n_.
  print 7.1 / 8.9 - 8.9, _n_.
  print 7.1 / 9.3 - 3.2, _n_.
  print 5.2 / 8.0 - 2.3, _n_.
  print 6.5 / 6.7 - 5.4, _n_.
  print 8.0 / 4.9 - 9.9, _n_.
  print 6.8 / 4.9 - 2.6, _n_.
  print 3.7 / 1.2 - 7.4, _n_.

  print 5.0 - 6.0 / 2.8 - 2.7, _n_.
  print 2.0 - 7.6 / 3.4 - 4.6, _n_.
  print 1.0 - 7.3 / 4.8 - 2.1, _n_.
  print 3.0 - 4.5 / 5.9 - 4.0, _n_.
  print 8.6 - 8.3 / 7.6 - 7.6, _n_.
  print 4.6 - 4.2 / 8.7 - 5.3, _n_.
  print 7.2 - 4.7 / 9.7 - 9.7, _n_.
  print 9.9 - 6.0 / 1.6 - 7.9, _n_.
  print 1.4 - 1.2 / 8.7 - 2.6, _n_.
  print 8.6 - 8.5 / 8.9 - 9.1, _n_.
  print 9.5 - 3.8 / 3.7 - 9.3, _n_.
  print 3.2 - 8.1 / 5.4 - 4.1, _n_.

  print 1.4 / 4.7 - 8.6 / 7.5, _n_.
  print 4.5 / 3.2 - 3.0 / 3.5, _n_.
  print 5.8 / 6.1 - 6.9 / 9.5, _n_.
  print 8.2 / 9.9 - 3.5 / 7.6, _n_.
  print 4.7 / 8.5 - 2.9 / 7.3, _n_.
  print 1.5 / 1.6 - 5.0 / 7.3, _n_.
  print 7.6 / 2.5 - 2.5 / 9.5, _n_.
  print 5.7 / 3.5 - 5.9 / 9.0, _n_.
  print 4.6 / 9.6 - 7.2 / 4.4, _n_.
  print 1.0 / 4.6 - 1.5 / 5.6, _n_.
  print 4.6 / 6.8 - 7.1 / 9.1, _n_.
  print 6.0 / 3.1 - 3.5 / 4.6, _n_.

}

