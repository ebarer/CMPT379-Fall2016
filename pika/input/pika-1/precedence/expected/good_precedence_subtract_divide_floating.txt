5.75714
5.4
6.67805
1.42143
8.2
7.34545
6.82623
2.85
1.1
6.16154
4.83678
0.772549
-5.80588
-4.4125
0.555556
-5.05814
-5.58571
-8.10225
-2.43656
-1.65
-4.42985
-8.26735
-1.21224
-4.31667
0.157143
-4.83529
-2.62083
-1.76271
-0.0921053
-1.18276
-2.98454
-1.75
-1.33793
-1.45506
-0.827027
-2.4
-0.848794
0.549107
0.224504
0.367757
0.155681
0.252568
2.77684
0.973016
-1.1572
-0.0504658
-0.103749
1.17461
